packages/css/src/utils/class-name-generator.ts

```diff
@@ -0,0 +1,39 @@
+const acceptPrefix = 'abcdefghijklmnopqrstuvwxyz_'.split('');
```

Comment:
Did you miss uppercase chars?

Reply:
Good point. I had a brief look at the spec. CSS class is case sensitive. We'll do more research to confirm it.